app/views/index.scala.html

```diff
@@ -0,0 +1,72 @@
+@(mapsUrl: String)
+<!DOCTYPE html>
+<html lang="[en]">
+    <head>
+        <meta charset="UTF-8">
+        <title>Minotaur</title>
+        <link rel="stylesheet" href="@routes.Assets.versioned("stylesheets/main.css")">
+        <link rel="shortcut icon" type="image/png" href="@routes.Assets.versioned("images/favicon.png")">
+        <script>
+                function loadMap() {
+                    // TODO The starting lat and long should come from the active DMA
+                    var cntrLatLong = {lat: 38.859540, lng: -77.332696};
+                    var map = new google.maps.Map(document.getElementById('map'), {
+                        zoom: 12,
+                        center: cntrLatLong,
+                        mapTypeId: google.maps.MapTypeId.HYBRID
+                    });
+                }
+        </script>
+        <script src="@mapsUrl&callback=loadMap" async defer> </script>
+    </head>
+    <body>
+        <div id="header">
+            <div class="group">
+                <a href="/" id="logo" class="group"><img alt="Trafficland_logo" src="@routes.Assets.versioned("images/trafficland_logo.gif")"/></a>
+            </div>
+        </div>
+        <div id="main-content">
+            <div id="dma-list" class="main-col">
+                <div class="dma-button-inactive">DMA0</div>
+                <div class="dma-button-inactive">DMA1</div>
+                <div class="dma-button-active">DMA2</div>
+                <div class="dma-button-inactive">DMA3</div>
+                <div class="dma-button-inactive">DMA4</div>
+                <div class="dma-button-inactive">DMA5</div>
+                <div class="dma-button-inactive">DMA6</div>
+                <div class="dma-button-inactive">DMA7</div>
+            </div>
+            <div id="active-dma" class="main-col">
+                <!-- TODO These images should be served by minotaur as a proxy to IE. -->
+                <div class="feed">
+                    <img src="http://tc01.bos.tl.com:8080/transcoder/frame.jpg?webid=1182&size=full" alt="1182">
```

Comment:
Are you planing on adding more to the feed div? If not, I would remove it and set the img's class to feed.